roles/node_images_management_vm/files/cloud/cloud.cfg.d/00_network.cfg

```diff
@@ -0,0 +1,20 @@
+network:
+  version: 2
+  ethernets:
+    # eth0: local interface, will receive DHCP from libvirt.
+    eth0:
+      dhcp4: true
+      dhcp6: false
+      mtu: 1500
+      addresses:
+        - 169.254.0.2/30
+    # eth1: metal interface, will receive a static IP.
+    eth1:
```

Comment:
eth0 and eth2 are true, eth1 is false.. just checking if intended.

Reply:
- eth0, `true` for local builds, and it doesn't hurt for production.
- eth1, `false` because this will always need a static IP set since it will serve the other hypervisors
- eth2, `true` because on a metal system this'll be the external nic and having DHCP function is helpful.